New blog post (#334)
astrolicious/studiocms: www/web/src/content/blog/whats-next-for-studiocms.md

---
title: Welcoming our latest maintainer!

> description: Welcoming our newest member, and paving the road to our dashboard overhaul.

Reviewer: ```suggestion
description: Welcoming our newest member, and dashboard!
```

---

> title: Whats next for StudioCMS?

Reviewer: ```suggestion
title: Welcoming our latest maintainer!
```

> description: New development team member, and dashboard project!

Reviewer: ```suggestion
description: Welcoming our newest member, and paving the road to our dashboard overhaul.
```

publishDate: 2024-10-03
ogVariant: fall
author: adam-matthiesen
---

> ## What's Next for StudioCMS?

Reviewer: ```suggestion
```

> Astro Studio is sunsetting, we’ve updated our branding, and we’re proud to announce our first sponsorship thanks to Turso. But what’s on the horizon for StudioCMS? We invite you to join us as we continue improving and expanding our CMS to meet the needs of our growing community.

Reviewer: ```suggestion
```
